### DISCUSSION TITLE
How to set the name for a Mac application

### DISCUSSION THREAD

**Discussion by hamada-applicita** [unoplatform/uno#15314] [Q&A]
We are using uno to build a mac catalyst application which we are distributing ourselves i.e. not through the appstore

After installing the .pkg file the application is always named in macos as **ProjectName.Mobile.app** and this seems to be coming from the name of the mobile csproj in the solution.

Ive tried setting
ApplicationTitle in the csproj file
CFBundleDisplayName in info.pList

But neither seems to affect the application name.

Is there any way to name the application without renaming the .csproj file?

**Comment by jeromelaban** [comment]
This particular feature is a bit obscure to manage as there are many hidden behaviors from macOS. For now, we do not have a documentation for this, but you can take the [Uno Calculator](https://github.com/unoplatform/calculator) particularly (https://github.com/unoplatform/calculator/pull/592, https://github.com/unoplatform/calculator/pull/593,  https://github.com/unoplatform/calculator/pull/590 and https://github.com/unoplatform/calculator/pull/591) as an example until we get the documentation updated.

**Comment by hamada-applicita** [comment]
Ok thankyou, i'll experiment with these and update here what solution I find that works best
